test/skimNtuple2018.cpp

```diff
@@ -2998,1401 +3370,1592 @@ int main (int argc, char** argv)
+  //        }
+
+          theSmallTree.m_bjet1_pt_raw = tlv_firstBjet_raw.Pt();
+          theSmallTree.m_bjet2_pt_raw = tlv_secondBjet_raw.Pt();
+
+          TLorentzVector tlv_bH_raw = tlv_firstBjet + tlv_secondBjet ;
+          vector <TLorentzVector> tlv_bH_raw_jetup(N_jecSources, tlv_bH_raw);
+          vector <TLorentzVector> tlv_bH_raw_jetdown(N_jecSources, tlv_bH_raw);
+
+          //JER: in LLR it is saved as jets_JER = JER (in %) * jet_energy:
+          //https://github.com/LLRCMS/LLRHiggsTauTau/blob/102X_HH/NtupleProducer/plugins/HTauTauNtuplizer.cc#L2316
+          double bjet1_JER = theBigTree.jets_JER->at(bjet1idx);
+          double bjet2_JER = theBigTree.jets_JER->at(bjet2idx);
+          theSmallTree.m_bjet1_JER = bjet1_JER;
+          theSmallTree.m_bjet2_JER = bjet2_JER;
+
+
+          //compute and store variations
+          for (int isource = 0; isource < N_jecSources; isource++)
+            {
+                // compute all shifted bjets
+              tlv_firstBjet_raw_jetup[isource]   = getShiftedJet(tlv_firstBjet_raw, +1., unc_first_updown.first[isource]);
+              tlv_firstBjet_raw_jetdown[isource] = getShiftedJet(tlv_firstBjet_raw, -1., unc_first_updown.second[isource]);
+              tlv_secondBjet_raw_jetup[isource]  = getShiftedJet(tlv_secondBjet_raw, +1., unc_second_updown.first[isource]);
+              tlv_secondBjet_raw_jetdown[isource]= getShiftedJet(tlv_secondBjet_raw, -1., unc_second_updown.second[isource]);
+
+
+              theSmallTree.m_bjet1_mass_raw_jetup.push_back(tlv_firstBjet_raw_jetup[isource].M());
+              theSmallTree.m_bjet1_mass_raw_jetdown.push_back(tlv_firstBjet_raw_jetdown[isource].M());
+              theSmallTree.m_bjet1_pt_raw_jetup.push_back(tlv_firstBjet_raw_jetup[isource].Pt());
+              theSmallTree.m_bjet1_pt_raw_jetdown.push_back(tlv_firstBjet_raw_jetdown[isource].Pt());
+              theSmallTree.m_bjet2_mass_raw_jetup.push_back(tlv_secondBjet_raw_jetup[isource].M());
+              theSmallTree.m_bjet2_mass_raw_jetdown.push_back(tlv_secondBjet_raw_jetdown[isource].M());
+              theSmallTree.m_bjet2_pt_raw_jetup.push_back(tlv_secondBjet_raw_jetup[isource].Pt());
+              theSmallTree.m_bjet2_pt_raw_jetdown.push_back(tlv_secondBjet_raw_jetdown[isource].Pt());
+              // variations propagated to bH
+              tlv_bH_raw_jetup.push_back(tlv_firstBjet_raw_jetup.at(isource) + tlv_secondBjet_raw_jetup.at(isource)) ;
+              tlv_bH_raw_jetdown.push_back(tlv_firstBjet_raw_jetdown.at(isource) + tlv_secondBjet_raw_jetdown.at(isource));
+              theSmallTree.m_bH_mass_raw_jetup.push_back((tlv_bH_raw_jetup[isource]).M());
+              theSmallTree.m_bH_mass_raw_jetdown.push_back((tlv_bH_raw_jetdown[isource]).M());
+              theSmallTree.m_bH_pt_raw_jetup.push_back((tlv_bH_raw_jetup[isource]).Pt());
+              theSmallTree.m_bH_pt_raw_jetdown.push_back((tlv_bH_raw_jetup[isource]).Pt());
```

Comment:
oops, this should be 
`theSmallTree.m_bH_pt_raw_jetdown.push_back((tlv_bH_raw_jetdpwn[isource]).Pt());`
(my mistake, I'll also fix it in skimNtuple2017).
Heads up to @francescobrivio (can't tag dzuolo), there's probably the same mistake in skimNtuple2016 if copied from my PR

Reply:
nice catch @camendola ok we will fix it on the 2016 PR as well